splice_machine/src/test/java/com/splicemachine/db/impl/sql/compile/StatsUsageIT.java

```diff
@@ -74,6 +74,31 @@ public static void createData(Connection conn, String schemaName) throws Excepti
                 "CALL SYSCS_UTIL.DISABLE_COLUMN_STATISTICS('%s','t1','b1')",
                 schemaName));
 
+        new TableCreator(conn)
+                .withCreate("create table t2 (a2 int, b2 int, c2 int)")
+                .withInsert("insert into t2 values(?,?,?)")
+                .withRows(rows(
+                        row(1,1,1),
+                        row(1,1,1),
+                        row(1,1,1),
+                        row(1,1,1),
+                        row(1,1,1),
+                        row(2,2,2),
+                        row(2,2,2),
+                        row(2,2,2),
+                        row(2,2,2),
+                        row(2,2,2)))
+                .create();
+
+        int factor = 10;
+        for (int i = 1; i <= 9; i++) {
+            spliceClassWatcher.executeUpdate(format("insert into t2 select a2+%d, b2,c2 from t2", factor));
+            factor = factor * 2;
+        }
+
+        conn.createStatement().executeUpdate(format("CALL SYSCS_UTIL.SYSCS_SPLIT_TABLE('%s', '%s')",
+                spliceSchemaWatcher.toString(), "T2"));
+
         conn.createStatement().executeQuery(format(
```

Comment:
After the syscs_split_table() call, there is a call to collect_schema_statistics()
        conn.createStatement().executeQuery(format(
                "call SYSCS_UTIL.COLLECT_SCHEMA_STATISTICS('%s',false)",
                schemaName));
which will collect stats on t2 also.